sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Release History
 
-## 4.24.0-beta.1 (Unreleased)
+## 4.24.0 (2024-08-19)
 
 ### Features Added
+- Added `StorageBlobAccessTier` metadata to `StorageBlobCreatedEventData` and `StorageBlobTierChangedEventData` [#41398](https://github.com/Azure/azure-sdk-for-java/pull/41398)
 
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Upgraded `azure-core-http-netty` from `1.15.2` to version `1.15.3`.
+- Upgraded `azure-core` from `1.50.0` to version `1.50.1`.
```

Comment:
The latest one is 1.51.0

https://mvnrepository.com/artifact/com.azure/azure-core

Reply:
good catch!